File: day-1/part-1/ruby/test_depth.rb
require_relative 'depth'
require 'test/unit'

class TestDepth < Test::Unit::TestCase
  def test_count_increase
    input = <<~TEXT
      199
      200
      208
      210
      200
      207
      240
      269
      260
      263
    TEXT

    assert_equal(7, Depth.new.count_increase(input))
  end
end

File: day-1/part-1/elixir/test_depth.exs
ExUnit.start()

defmodule DepthTest do
  use ExUnit.Case

  test "count_increase" do
    input = """
      199
      200
      208
      210
      200
      207
      240
      269
      260
      263
    """
    assert Depth.count_increase(input) == 7
  end
end
